tests/installation/setup_online_repos.pm

```diff
@@ -24,6 +24,10 @@ sub run {
     if (is_leap('<15.1')) {
         @default_repos = qw(update-non-oss update-oss main-non-oss main-oss debug-main untested-update debug-update source);
     }
+    elsif (is_leap('>=15.3')) {
+        # 15.3 has introduced additional repositories
+        @default_repos = qw(update-sle update-backports update-non-oss main-non-oss update-oss main-oss debug-backports-update debug-update untested-update source debug-main);
+    }
```

Comment:
I suggest to only add repos to `@default_repos`, not define the array differently

Reply:
my original idea was add repos to `@default_repos` for sure until I realized it has a certainly ordering per real reposlist, that means `debug-backports-update` have to insert to between `main-oss` and `debug-update`, then the code looks ugly even.